front/src/modules/trainschedule/components/TimetableV2/hooks.ts

```diff
@@ -146,6 +164,7 @@ const useTrainSchedulesDetails = (
                   duration: trainSummary.time,
                   pathLength: formatKmValue(trainSummary.length, 'millimeters', 1),
                   mechanicalEnergyConsumed: jouleToKwh(trainSummary.energy_consumption, true),
+                  scheduledPointsHonored: trainSummary.scheduled_points_honored,
```

Comment:
Maybe it would be better to use the contrary, this way, in the TimetableTrainCardV2, you can display the yellow band only if `scheduledPointsNotHonored` is true. It will solve the problem that a invalid train has the yellow band
```suggestion
                  scheduledPointsNotHonored: trainSummary.scheduled_points_honored,
```

Reply:
Done